cfgov/v1/models/atoms.py

```diff
@@ -1,6 +1,70 @@
+from django.core.exceptions import ValidationError
+
 from wagtail.wagtailcore import blocks
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+
+
+def isRequired(field_name):
+    return [str(field_name) + ' is required.']
 
 
 class Hyperlink(blocks.StructBlock):
-    label = blocks.CharBlock(max_length=50)
-    href = blocks.CharBlock(default='/')
+    text = blocks.CharBlock(max_length=50, required=False)
```

Comment:
huh? they are all changed in the molecule templates that refer to links. can you point one out where your seeing the change not made.